web-wallet/src/lib/components/__tests__/Stake.spec.js

```diff
@@ -235,4 +235,108 @@ describe("Stake", () => {
       expect(() => getByRole("link", { name: /explorer/i })).toThrow();
     });
   });
+
+  describe("Unstake operation", () => {
+    const expectedExplorerLink = `https://explorer.dusk.network/transactions/transaction?id=${lastTxId}`;
+
+    afterAll(() => {
+      vi.useRealTimers();
+    });
+
+    it("should perform an ustake, give a success message and supply a link to see the transaction in the explorer", async () => {
+      vi.useFakeTimers();
```

Comment:
If all the tests in the group need fake timers, then I think it's better to have `vi.useFakeTimers()` just after the `describe` of the group, instead of repeating that for every single test.

Reply:
I had done it like this, but it didn't work for some reason...

<img width="1802" alt="Screenshot 2024-07-26 at 11 27 54" src="https://github.com/user-attachments/assets/2570aa86-1fb4-4cff-9db7-9283ed21869e">